bls/src/main/java/tech/pegasys/teku/bls/BLSSecretKey.java

```diff
@@ -79,4 +74,9 @@ public boolean equals(final Object o) {
   public int hashCode() {
     return Objects.hash(secretKey);
   }
+
+  @Override
+  public String toString() {
+    return "BLSSecretKey{" + toBytes() + '}';
```

Comment:
Do we need this?  Seems potentially dangerous ...

Reply:
Yes, agree, it's dangerous indeed